### DISCUSSION TITLE
Horizontal Buttons Stack: Customizing color only for pop up links

### DISCUSSION THREAD

**Discussion by christianvay** [Clooos/Bubble-Card#306] [Feature requests]
I want to use the horizontal button stack as a fixed room navigation bar in the footer. I created a button for each room and put in the link to the page in my dashboard. I have a copy of this stack on each of my pages so I can easily navigate.
But now I wanted to customize the buttons. I started with color and figured out, that I can only change the color for buttons which links to a pop up but not to the ones which are linked to another page.

Link to pop up - color changed:

type: custom:bubble-card
card_type: horizontal-buttons-stack
1_name: test-kuche
1_icon: mdi:home
1_link: '#test-kuche'
styles: |
  .test-kuche {
    background: red !important;
    border: 10px solid darkblue !important;
  }

Link to page - color not changed:

type: custom:bubble-card
card_type: horizontal-buttons-stack
1_name: test-kuche
1_icon: mdi:home
1_link: test-kuche
styles: |
  .test-kuche {
    background: red !important;
    border: 10px solid darkblue !important;
  }

Any idea how to change color for linked to pages buttons? Beside this issue it is a great card set. Thank you for that.

Regards,
Christian

**Comment by dnestico** [comment]
Seeing this too now, using links instead of a hash but the button color won't change now.
